test/test_session.cpp

```diff
@@ -510,3 +510,70 @@ TORRENT_TEST(init_dht)
 		TEST_EQUAL(count, 1);
 	}
 }
+
+TORRENT_TEST(reopen_network_sockets)
+{
+	auto count_alerts = [](session& ses, int listen, int portmap)
```

Comment:
both ``listen`` and ``portmap`` can be const

Reply:
OK